### PR TITLE
FOUR-10042: HOME: Add in the component "COUNT"

### PR DESCRIPTION
**Pull request by CarliPinell**
## Issue & Reproduction Steps
Element "Count" was added to new control List Table
## Related Tickets & Packages
- FOUR-10042

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.
ci:next

### CODE REVIEW

File: src/components/renderer/form-list-table.vue [hunk 1]
```diff
@@ -1,20 +1,42 @@
 <template>
-  <div class="card mt-4 mb-4">
-    <div class="card-header d-flex justify-content-between align-items-center">
-      <h4>{{ title }}</h4>
-      <div>
-        <i class="fas fa-search" />
+  <div class="container mt-4">
```
Comment (fagubla) [inline src/components/renderer/form-list-table.vue:2]: remove this div

File: src/components/renderer/form-list-table.vue [hunk 2]
```diff
@@ -1,20 +1,42 @@
...
+    <div class="card">
```
Comment (fagubla) [inline src/components/renderer/form-list-table.vue:3]: add mt-4 and mb-4

File: src/components/renderer/form-list-table.vue [hunk 3]
```diff
@@ -31,27 +53,53 @@ export default {
...
+    }
+  }
 };
 </script>
 
 <style lang="scss">
 .prevent-interaction.form-list-table::after {
   content: attr(placeholder);
 }
-.list-table {
```
Comment (fagubla) [inline src/components/renderer/form-list-table.vue:53]: don't remove this css

File: src/components/renderer/form-new-request.vue [hunk 4]
```diff
@@ -77,6 +77,13 @@ export default {
           data.meta.from -= 1;
           this.$refs.listProcess.data = data;
           this.$refs.listProcess.setPaginationData(data.meta);
+          const dataStart = {
```
Comment (fagubla) [inline src/components/renderer/form-new-request.vue:80]: for this control the counter is not required

File: src/components/renderer/form-requests.vue [hunk 5]
```diff
@@ -110,6 +111,16 @@ export default {
               record.status = this.formatStatus(record.status);
             }
             this.tableData = response.data;
+            this.countResponse = Object.keys(this.tableData.data).length;
```
Comment (fagubla) [inline src/components/renderer/form-requests.vue:114]: the code is duplicate in two file, create a mixin file

**Review by fagubla — CHANGES_REQUESTED** [review]

File: src/components/renderer/form-list-table.vue (hunk 3 above)
Comment (CarliPinell) [reply src/components/renderer/form-list-table.vue:53]: solved

File: src/components/renderer/form-list-table.vue (hunk 1 above)
Comment (CarliPinell) [reply src/components/renderer/form-list-table.vue:2]: solved.

File: src/components/renderer/form-list-table.vue (hunk 2 above)
Comment (CarliPinell) [reply src/components/renderer/form-list-table.vue:3]: solved.

File: src/components/renderer/form-new-request.vue (hunk 4 above)
Comment (CarliPinell) [reply src/components/renderer/form-new-request.vue:80]: As we agree with Fabio explaining this code, this counter is needed and is only applied to tasks and requests to obtain the number of rows returned in Api response to emit the event to father component.

File: src/components/renderer/form-requests.vue (hunk 5 above)
Comment (CarliPinell) [reply src/components/renderer/form-requests.vue:114]: Each count is related to its own component, a mixin file is not needed.

**Review by fagubla — APPROVED** [review]

**Pull request merged**